src/tritonserver.cc

```diff
@@ -1692,6 +1692,25 @@ TRITONSERVER_InferenceRequestSetCorrelationIdString(
 TRITONAPI_DECLSPEC TRITONSERVER_Error*
 TRITONSERVER_InferenceRequestPriority(
     TRITONSERVER_InferenceRequest* inference_request, uint32_t* priority)
+{
+  uint64_t temp;  
+  auto error = TRITONSERVER_InferenceRequestPriorityU64(inference_request, &temp);
+  if (error != nullptr) {
+    return error;
+  }
+  if (temp <= std::numeric_limits<uint32_t>::max()) {
+    *priority = temp;
+    return nullptr; // Success
+  }
+  return TRITONSERVER_ErrorNew(TRITONSERVER_ERROR_INVALID_ARG,
+			       (std::string(
+				   "request priority overflows uint32_t, priority=") +
```

Comment:
Perhaps updating to error message to tell the user to use "TRITONSERVER_InferenceRequestPriorityU64" instead.

Reply:
will update when name is settled